sources/web/datalab/static/datalab.css

```diff
@@ -115,13 +115,15 @@ body {
   justify-content: space-between;
 }
 #contentArea {
-  height: 100%;
```

Comment:
Interesting, it looks fine for me. I'm wondering what else am I missing when I test UI on my Chrome/Firefox then..